style.css

```diff
@@ -0,0 +1,149 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@400;700&family=Poppins:wght@400;600&display=swap');
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  text-decoration: none;
+  font-family: Inter, sans-serif;
+}
+
+body {
+  background-image: url("header-illsutration-mobile.svg");
+  background-size: 100%;
+  background-repeat: no-repeat;
+  background-position: 10px 60px;
+  width: 90%;
+  margin: 0;
+  font-family: Inter, sans-serif;
+}
+
+nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  color: white;
+  height: 50px;
+  width: auto;
+}
+
+nav ul {
+  margin: 0;
+  padding: 0;
+  display: flex;
+}
+
+nav li {
+  list-style: none;
+}
+
+nav li a {
+  text-decoration: none;
+  color: black;
+  padding: 1rem;
+  display: block;
+}
+
+nav li:hover {
+  background-color: aqua;
+}
+
+.toggle-button {
+  position: absolute;
+  top: 1.8rem;
+  right: 1rem;
+  color: black;
+  display: none;
+  flex-direction: column;
+  justify-content: space-between;
+  width: 30px;
+  height: 21px;
+  line-height: 28px;
+  margin: 6px 20px;
+}
+
+.toggle-button .bar {
+  height: 3px;
+  width: 100;
+  background-color: black;
+  border-radius: 10px;
+}
+
+@media (max-width: 768px) {
+  .toggle-button {
+    display: flex;
+  }
+
+  nav ul {
+    display: none;
+  }
+}
+
+.hero h1 {
+  color: #172b4d;
+  font-family: Inter, sans-serif;
+  top: 156px;
+  font-size: 40px;
+  font-weight: 700;
+  font-style: normal;
+  text-align: center;
+  line-height: 52px;
+  margin: 110px 0 0 0;
+}
+
+.hero span {
+  color: #36b37e;
+}
+
+.hero p {
+  color: #172b4d;
+  font-family: Inter, sans-serif;
+  font-weight: 400;
+  font-size: 19px;
+  line-height: 28px;
+  height: 164px;
+  text-align: center;
+  left: 20px;
+  margin: 20px 0 0 0;
+  padding-left: 30px;
+}
+
+.social-links {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+  padding: 0;
+  text-align: center;
+  position: absolute;
+  bottom: 0;
+  height: 250px;
+  left: 50%;
+  transform: translateX(-50%);
+}
+
+.social-links ul li {
+  display: inline-block;
+  margin: 0 10px;
+}
+
+.social-links li:hover {
+  background: aqua;
+}
+
+@media (max-width: 768px) {
+  .social-links {
+    width: 100%;
+    margin: -60px 12px;
+  }
+}
+
+.arrow-container {
+  position: fixed;
+  bottom: 0;
+  width: 100%;
+  height: 50px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
```

Comment:
- [OPTIONAL] You made this arrow icon fixed whereas in the Figma template, it is not fixed but positioned beneath the social media icons. I would advise you to position it properly and not make it fixed. 👍